docs/revsets.md

```diff
@@ -211,8 +212,9 @@ revsets (expressions) as arguments.
 * `none()`: No commits. This function is rarely useful; it is provided for
   completeness.
 
-* `change_id(prefix)`: Commits with the given change ID prefix. It is an error
-  to use a non-unique prefix. Unmatched prefix isn't an error.
+* `change_id(prefix)`: Commits with the given change ID prefix. If the specified
+  change is diverged, this resolves to multiple commits. It is an error to use a
```

Comment:
nit: I think "is diverged" (here and elsewhere) should be either "is divergent" or "has diverged". I think the former is better since it matches the template function. We also seem to use that form much more frequently in our docs.

Reply:
Updated to "divergent"